### PR TITLE
Adding rubygems_url setting documentation for air gapped environments

### PR DESCRIPTION
**Pull request by iennae**
Resolves Issues 
#206
https://github.com/chef-cookbooks/rubygems/issues/15

### CODE REVIEW

**Comment by iennae** [comment]
@coderanger can you take a peek at this. Not sure if I should add any additional information with regards to air gapped environments or if that should be a separate document somewhere.

**Review by coderanger — APPROVED** [review]
This is good enough for the moment. Long term it should also be mentioned in a narrative section in https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_chef_gem.rst because that's probably where people will come in looking for "how do I use chef_gem with my own gems server".

**Comment by coderanger** [comment]
I am reminded while we're talking on Slack, this should mention the config option is new in 13.0.

File: chef_master/source/config_rb_client.rst [hunk 1]
```diff
@@ -300,6 +300,11 @@ This configuration file has the following settings:
 ``role_path``
    The location in which role files are located. Default value: ``/var/chef/roles``.
 
+``ruby_gem_url``
```
Comment (coderanger) [inline chef_master/source/config_rb_client.rst]: Should be `rubygems_url`.

**Comment by davidwrede** [comment]
tenor-184674784

**Pull request merged**